x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/bulk_update_rules/route.ts

```diff
@@ -69,51 +71,86 @@ export const bulkUpdateRulesRoute = (
         savedObjectsClient,
       });
 
+      const ruleDefinitions = request.body;
+
+      const rulesIndexesWithDuplicatedDefaultExceptionsList =
+        getRulesIndexesWithDuplicatedDefaultExceptionsList(ruleDefinitions);
+
       const rules = await Promise.all(
-        request.body.map(async (payloadRule) => {
-          const idOrRuleIdOrUnknown = payloadRule.id ?? payloadRule.rule_id ?? '(unknown id)';
-          try {
-            const validationErrors = validateUpdateRuleProps(payloadRule);
-            if (validationErrors.length) {
-              return createBulkErrorObject({
+        ruleDefinitions
+          .filter((rule, index) => !rulesIndexesWithDuplicatedDefaultExceptionsList.includes(index))
```

Comment:
I suggest removing this filter and returning a `createBulkErrorObject` from the map body instead.